src/components/SearchBar.js

```diff
@@ -16,69 +16,87 @@ import Signals from '../api/Signals';
 declare var webkitSpeechRecognition: any; // Prevent complaints about this not existing
 
 type SearchBarProps = {
-  history: PropTypes.object.isRequired;
+  history: PropTypes.object.isRequired,
   /** If set, this will be styled to live inside a Masthead component. */
-  inMasthead: boolean;
+  inMasthead: boolean,
```

Comment:
For reference, these booleans have defaults set and should also indicate they are optional, so they aren't a good reference for best practices. This one should be:
`inMasthead?: boolean,`